src/main/java/seedu/address/storage/JsonAddressBookStorage.java

```diff
@@ -21,19 +22,32 @@ public class JsonAddressBookStorage implements AddressBookStorage {
 
     private static final Logger logger = LogsCenter.getLogger(JsonAddressBookStorage.class);
 
-    private Path filePath;
+    private Path personsFilePath;
 
-    public JsonAddressBookStorage(Path filePath) {
-        this.filePath = filePath;
+    private Path appointmentFilePath;
+
+    /**
+     * Creates a {@code JsonAddressBookStorage} with the given file paths.
+     * @param personsFilePath the file path for the person data. Cannot be null.
+     * @param appointmentFilePath the file path for the appointment list data. Can be null.
+     */
+
+    public JsonAddressBookStorage(Path personsFilePath, Path appointmentFilePath) {
+        this.personsFilePath = personsFilePath;
+        this.appointmentFilePath = appointmentFilePath;
+    }
+
+    public JsonAddressBookStorage(Path personsFilePath) {
```

Comment:
Do you think we should remove this constructor? I was thinking we can actually remove it, but I am not sure if it is worth the time / effort?

Reply:
yeah need to rerun some test case